deps/noise/noise.c

```diff
@@ -123,6 +126,7 @@ void seed(unsigned int x) {
         PERM[i] = b;
         PERM[j] = a;
     }
+   
     memcpy(PERM + 256, PERM, sizeof(unsigned char) * 256);
```

Comment:
If the size of PERM array is changing to 255, does the bound on the memcpy here need to change as well?